modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -0,0 +1,320 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-cluster-customizations-ocm_{context}"]
+= Creating a cluster through {cluster-manager} using customizations
+
+When you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively through {cluster-manager-first}.
+
+You can also choose between the `auto` and `manual` modes when creating the required STS resources.
+
+[IMPORTANT]
+====
+Only public and AWS PrivateLink clusters are supported with STS. Regular private clusters (non-PrivateLink) are not available for use with STS.
+====
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installations.
+====
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
++
+[NOTE]
+====
+To successfully install ROSA 4.10 clusters, use ROSA CLI 1.1.11 or above.
```

Comment:
Specifically, we actually want 1.2.2 (to be released next week). Prior to 1.2.0, the ROSA CLI defined the policy for the OCM and user roles at its end without fetching the policies from CS. As of 1.2.0, the ROSA CLI fetches the latest policies for these roles from CS and the latest policies are required to successfully create ROSA clusters.

Reply:
Thank you for highlighting this. As per our recent Slack conversation on this topic, I have replaced the note and other notes like it in the PR with the following:

> **Note**: To successfully install ROSA 4.10 clusters, use latest version of the ROSA CLI.

This will ensure that the note does not become stale in the future.

Reply:
For reference, I previously created and [SDA-5624 - Request to add the ROSA CLI version number to the OCM UI](https://issues.redhat.com/browse/SDA-5624), which I think will help cluster administrators easily see if a later version of the CLI is available.

*